base/src/EglRenderer.cpp

```diff
@@ -38,18 +39,27 @@ class EglRenderer::Detail
         return true;
     }
 
+    bool destroyWindow()
+    {
+        if(renderer)
+        {
+            delete renderer;
+        }
+    }
+
 	bool shouldTriggerSOS()
 	{
 		return !renderer;
 	}
 
 	NvEglRenderer *renderer = nullptr;
     uint32_t x_offset,y_offset,width,height;
+    bool displayOnTop;
```

Comment:
I don't think we are using displayOnTop variable anywhere

Reply:
we are using it here displayOnTop(_displayOnTop) in detail